controllers/flowcollector_controller.go

```diff
@@ -74,6 +77,14 @@ func Start(ctx context.Context, mgr *manager.Manager) error {
 	}
 	r.watcher = watchers.NewWatcher(ctrl)
 
+	reconcileAfter := func() {
+		// force reconcile at startup
+		time.Sleep(30 * time.Second)
+		_, err := ctrl.Reconcile(ctx, reconcile.Request{})
+		log.Error(err, "Error while doing initial reconcile")
+	}
+	go reconcileAfter()
```

Comment:
:point_up: I wonder if there is an out of box mechanism to trigger the loop after the cache loaded. That's why I'm using a sleep here and this will may work in all situations.

https://redhat-internal.slack.com/archives/C02939DP5L5/p1743518264445429